gratipay/utils/markdown.py

```diff
@@ -19,6 +19,13 @@ def render(markdown):
     ))
 
 
+def rtlo_scrub(markdown):
```

Comment:
I can, but if we somehow want to support RTLO languages in the future this may actually cause some issues, therefore to make it easy to spot and resolve I have made it into a separate function. Would you still prefer that I place everything into one function?